modules/assim.sequential/inst/anchor/anchorSites_data_prep.R

```diff
@@ -0,0 +1,74 @@
+library(dplyr)
+library(xts)
+library(PEcAn.all)
+library(purrr)
+library(furrr)
+library(lubridate)
+library(nimble)
+library(ncdf4)
+library(PEcAnAssimSequential)
+library(dplyr)
+library(sp)
+library(raster)
+library(zoo)
+library(ggplot2)
+library(mnormt)
+library(sjmisc)
+library(stringr)
+library(doParallel)
+library(doSNOW)
+library(Kendall)
+library(lgarch)
+library(parallel)
+if (future::supportsMulticore()) {
+  future::plan(future::multicore)
+} else {
+  future::plan(future::multisession)
+}
+
+setwd("/projectnb/dietzelab/dongchen/anchorSites/")
+settings <- PEcAn.settings::read.settings("/projectnb/dietzelab/dongchen/anchorSites/SDA/pecan.xml")
+
+in.path <- "/projectnb/dietzelab/hamzed/ERA5/Data/Ensemble"
+out.path <- "/projectnb/dietzelab/dongchen/anchorSites//ERA5_2012_2021"
```

Comment:
Since this is a stand-alone script, I think it would be good to include an overall introductory comment block explaining what the purpose of the script is and how to use it, and then additional comments on variables like these (settings, in.path, out.path) explaining what they should be.  

Also, is there something here that's specific to anchor sites or would this script do data prep for any set of sites (in North America? in the world?) and the initial list of sites just happens to be anchor sites?

Reply:
1) I replaced it with well-documented Rmd file.
2) It's preparing data for the 342 NA anchor sites that combine the Bety site group (1000000033) and some sites in the goolge sheet (https://docs.google.com/spreadsheets/d/1n7pVUcrYrB0S8bqrj77tUNrLHA2c_yqzkZL8mgdVDjs/edit#gid=0). I also left comments for this information.